src/gateways/logEvent.js

```diff
@@ -0,0 +1,18 @@
+import fetch from "isomorphic-unfetch";
+
+const logEvent = (key, url) => async (event) => {
+  const body = JSON.stringify(event);
+  const response = await fetch(url, {
+    method: "POST",
+    body: body,
+    headers: {
+      "x-functions-key": key,
+    },
+  }).catch((error) => {
+    console.error("Error (log event):", error);
+    console.log(`the url is ${url}`);
+  });
```

Comment:
Use the `try {} catch (){}` pattern here if you can, it ends up being a bit cleaner than dealing with promise callbacks especially given you're already making use of `await` here rather than `.then()`